index.js

```diff
@@ -44,48 +45,85 @@ function cleanClock (clock) {
 }
 
 exports.init = function (sbot, config) {
-  const dir = config.path ? path.join(config.path, 'ebt') : null
-  const store = Store(dir, null, toUrlFriendly)
-
-  const ebt = EBT({
-    logging: config.ebt && config.ebt.logging,
-    id: sbot.id,
-    getClock (id, cb) {
-      store.ensure(id, function () {
-        const clock = store.get(id) || {}
-        cleanClock(clock)
-        cb(null, clock)
-      })
-    },
-    setClock (id, clock) {
-      cleanClock(clock, 'non-feed key when saving clock')
-      store.set(id, clock)
-    },
-    getAt (pair, cb) {
-      sbot.getAtSequence([pair.id, pair.sequence], (err, data) => {
-        cb(err, data ? data.value : null)
-      })
-    },
-    append (msg, cb) {
-      sbot.add(msg, (err, msg) => {
-        cb(err && err.fatal ? err : null, msg)
-      })
-    },
-    isFeed: isFeed
-  })
+  const formats = {
+    'classic': require('./formats/classic')
+  }
+
+  const ebts = {}
+  function addEBT(formatName) {
```

Comment:
Would be rather easy to do `addEBT(formatName, methods)` or `addEBT(formatName, format)` here and then we don't need to use the global `formats` here. It's good to reduce usage of globals if it's easy, makes this function more refactorable/movable in the future.

Reply:
Yeah this is a good idea, latest changes hopefully makes these things clearer